lib/commonio.c

```diff
@@ -354,33 +355,25 @@ bool commonio_present (const struct commonio_db *db)
 
 int commonio_lock_nowait (struct commonio_db *db, bool log)
 {
-	char* file = NULL;
-	char* lock = NULL;
-	size_t lock_file_len;
-	size_t file_len;
-	int err = 0;
+	int   err = 0;
+	char  *file = NULL;
+	char  *lock = NULL;
 
 	if (db->locked) {
 		return 1;
 	}
-	file_len = strlen(db->filename) + 11;/* %lu max size */
-	lock_file_len = strlen(db->filename) + 6; /* sizeof ".lock" */
-	file = MALLOC(file_len, char);
-	if (file == NULL) {
+
+	if (asprintf(&file, "%s.%lu", db->filename, (unsigned long) getpid()) == -1)
```

Comment:
Use key characters `{}` at the beginning and end of the condition. I know it's only one line of condition and there's no need for it, but I've had very bad experiences when forgetting these characters. The last one was ten days ago, when a colleague didn't understand didn't understand why his code was failing and he spent more than one hour due to this problem.

Reply:
IMO braces are not needed, the problem is simply that the if line is > 80 chars.

If you did something like

```
serge@jerom ~/src/shadow$ git diff
diff --git a/lib/commonio.c b/lib/commonio.c
index 6f950edef..9829fa850 100644
--- a/lib/commonio.c
+++ b/lib/commonio.c
@@ -356,6 +356,7 @@ bool commonio_present (const struct commonio_db *db)
 int commonio_lock_nowait (struct commonio_db *db, bool log)
 {
        int   err = 0;
+       unsigned long p = getpid()
        char  *file = NULL;
        char  *lock = NULL;

@@ -363,7 +364,7 @@ int commonio_lock_nowait (struct commonio_db *db, bool log)
                return 1;
        }

-       if (asprintf(&file, "%s.%lu", db->filename, (unsigned long) getpid()) == -1)
+       if (asprintf(&file, "%s.%lu", db->filename, p) == -1)
                goto cleanup_ENOMEM;
        if (asprintf(&lock, "%s.lock", db->filename) == -1)
                goto cleanup_ENOMEM;
```
it would IMO be crystal clear.

Reply:
I see you two see this point in the same way. So @hallyn do we have any rule for brackets? If not, I'll withdraw this comment.

Reply:
Yeah, [doc/contributions/coding_style.md](https://github.com/shadow-maint/shadow/blob/master/doc/contributions/coding_style.md) points to the kernel coding style, which has:

https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tree/Documentation/process/coding-style.rst?h=v6.6-rc4#n197